backend/routes/auth.js

```diff
@@ -570,6 +572,8 @@ function createSession(place, req, res, next, user) {
 			return removeSessions(ids);
 		}).then(() => {
 			responseCodes.respond(place, req, res, next, responseCodes.OK, user);
+		}).then(()=>{
```

Comment:
This would imply that you would only want to save the record after you respond. We don't need this to be done synchronously.

I would probably put it on L556

```` js 
.then(() => { 
   LoginRecord.saveLoginRecord(req);
   return getSessionsByUsername(user.username));
}
````